package.json

```diff
@@ -24,7 +24,7 @@
   },
   "types": "./dist/widgets.d.ts",
   "private": true,
-  "version": "1.18.1",
+  "version": "1.19.0",
```

Comment:
should we bump the version here or in a separate PR/commit?

Reply:
I did a [PR](https://github.com/cowprotocol/cowswap/pull/824) for this